packages/libcap.rb

```diff
@@ -2,20 +2,12 @@
 
 class Libcap < Package
   description 'Libcap implements the user-space interfaces to the POSIX 1003.1e capabilities available in Linux kernels.'
-  homepage 'https://directory.fsf.org/wiki/Libcap'
-  version '2.66'
+  homepage 'https://directory.fsf.org/wiki/Libcap/'
+  version '2.70'
   license 'GPL-2 or BSD'
   compatibility 'all'
-  source_url 'https://git.kernel.org/pub/scm/libs/libcap/libcap.git'
+  source_url 'https://kernel.googlesource.com/pub/scm/libs/libcap/libcap.git'
```

Comment:
Any particular reason to swap to google's mirror? It requires a google account to see the gitweb, which isn't great.